lib/perl/Genome/Site/TGI/Extension/Sys.pm

```diff
@@ -128,21 +126,19 @@ sub _bsub_and_wait_for_completion__validate_args {
     return(\%special_args, %bsub_args);
 }
 
-sub _bsub_and_wait_for_completion__wait_on_jobs {
-    my($class, $waiting_socket, $on_complete_cb, %seq_to_job_id) = @_;
-
-    while (%seq_to_job_id) {
-        my $fh = $waiting_socket->accept();
-        my $seq = $fh->getline;
-        $fh->close;
-        $seq =~ s/\r|\n//g;
-
-        my $job_id = delete $seq_to_job_id{$seq};
-        unless ($job_id) {
-            Carp::croak("Got unexpected response '$seq' while waiting for these jobs:\n",
-                join("\n", map { join(' => ', $_, $seq_to_job_id{$_}) } keys %seq_to_job_id));
+sub _bsub_and_wait_for_completion__wait_on_jobs_with_lsf_dependency {
+    my ( $class, $seq_to_job_id, $bsub_args, $on_complete_cb ) = @_;
+    local $ENV{LSF_NIOS_JOBSTATUS_INTERVAL} = 1;
```

Comment:
Setting it in the docker container makes sense, since it's required to make this succeed in the docker container.  I don't think setting it here makes sense, since it's not necessary outside of docker (or possibly even in the future when we're granted better support for opening ports).